lib/server.js

```diff
@@ -13,10 +13,12 @@ const causes = require('./routes/causes')
 const events = require('./routes/events')
 const localities = require('./routes/localities')
 const records = require('./routes/records')
+const auth = require('./auth')
```

Comment:
Let's move this up to line 2 of this file, so it's not mixed in with all of the routes.